opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/view.py

```diff
@@ -15,17 +15,36 @@
 
 from fnmatch import fnmatch
 from logging import getLogger
-from typing import Optional, Set, Type
+from typing import Callable, Optional, Set, Type
 
 from opentelemetry.metrics import Instrument
 from opentelemetry.sdk.metrics._internal.aggregation import (
     Aggregation,
     DefaultAggregation,
+    _Aggregation,
+    _ExplicitBucketHistogramAggregation,
+    _ExponentialBucketHistogramAggregation,
+)
+from opentelemetry.sdk.metrics._internal.exemplar import (
+    AlignedHistogramBucketExemplarReservoir,
+    ExemplarReservoirFactory,
+    SimpleFixedSizeExemplarReservoir,
 )
 
 _logger = getLogger(__name__)
 
 
+def default_reservoir_factory(
```

Comment:
Nit: Make private.

Reply:
I set it public because I had in the mind the following pattern for customization:

```py
def my_reservoir_factory(aggregationType) -> ExemplarReservoirBuilder:
    if my_condition is True:
        return MyCustomReservoir
    else:
        default_reservoir_factory(aggregationType)
```

Should I revert it to public?

Reply:
For our first roll out of the exemplars sdk we can make it private and if there is more demand for customization we can explore making it public. We just want to keep our api surface as small as possible. Not a strong opinion though.